Add documentation for localstorage (#1629)
cocos-creator/creator-docs: en/advanced-topics/data-storage.md

var userData = JSON.parse(cc.sys.localStorage.getItem('userData'));
```

## Clear data

> We can clear the data from the storage when we don't need it anymore:

Reviewer: ```suggestion
Data can be cleared from storage when it is no longer needed:
```
